app.rb

```diff
@@ -21,5 +22,18 @@ class RockPaperScissors < Sinatra::Base
     erb(:play)
   end
 
+  post "/move" do
+    session[:move] = params[:move]
+    redirect "/outcome"
+  end
+
+  get "/outcome" do
+    @move = session[:move]
+    @player_name = session[:player_name]
+    computer = Computer.new
```

Comment:
This makes so much sense!! Really clean, concise implementation of another class